fabric-renderer-registries-v1/src/main/java/net/fabricmc/fabric/impl/client/renderer/registry/ItemHooks.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.client.renderer.registry;
+
+import net.fabricmc.fabric.api.client.rendereregistry.v1.ItemOverlayRenderer;
+
+public interface ItemHooks {
```

Comment:
Shouldn't this be in impl package?

Reply:
Maybe specify it as `ItemOverlayExtensions` so we don't have multiple interfaces of the same name.

Reply:
I *swear* this was in the impl package

Reply:
Must have misread cause it is

Reply:
it *is* in the impl package my guy
I'll rename it tho